packages/contract-addresses/src/index.ts

```diff
@@ -19,34 +30,34 @@ export enum NetworkId {
 
 const networkToAddresses: { [networkId: number]: ContractAddresses } = {
     1: {
```

Comment:
This could probably be cleaned up by just iterating through the enum and building the object. Or is there something else upcoming planned which would prevent this?

```typescript
[1, 3, 42].reduce((acc, i) => {
    return {
        ...acc,
        [i]: {
            erc20Proxy: (ERC20Proxy as ContractArtifact).networks[i].address,
            erc721Proxy: (ERC721Proxy as ContractArtifact).networks[i].address,
            zrxToken: (ZRXToken as ContractArtifact).networks[i].address,
            etherToken: (WETH9 as ContractArtifact).networks[i].address,
            exchange: (Exchange as ContractArtifact).networks[i].address,
            assetProxyOwner: (AssetProxyOwner as ContractArtifact).networks[i].address,
            forwarder: (Forwarder as ContractArtifact).networks[i].address,
            orderValidator: (OrderValidator as ContractArtifact).networks[i].address,
        },
    };
}, {});
```